server/__init__.py

```diff
@@ -40,6 +40,10 @@
 
 propagator = google_cloud_format.GoogleCloudFormatPropagator()
 
+nl_model_cache_key = 'nl_model'
+nl_model_cache_path = '~/.datacommons/'
+nl_model_cache_expire = 3600  # Cache for 1 hour
```

Comment:
Could we make this last longer? Like 24*3600? The model inputs don't change often. And if they do, one can always empty the cache.

Reply:
Done